static/js/biomedical/disease/page.tsx

```diff
@@ -72,6 +73,7 @@ export interface PageStateType {
   diseaseCommonName: string;
   diseasePrevalenceIDexists: boolean;
   dataFetched: boolean;
+  diseaseParent: [];
```

Comment:
instead of just array, please include the type of objects in the array. e.g., `NamedNode[]`